lib/teiserver_web/live/battles/match/show.ex

```diff
@@ -57,7 +57,7 @@ defmodule TeiserverWeb.Battle.MatchLive.Show do
     # Restrict the balance tab to certain roles.
     # Note that Staff roles like "Tester" also contain Contributor role.
```

Comment:
The comment should be changed now that Contributor contains Tester and not the other way.

Reply:
Have updated in last commit